src/concepts/neuroscience.md

```diff
@@ -0,0 +1,102 @@
+
++ DataJoint Workflow Tools:
+  + element-miniscope: [https://github.com/datajoint/element-miniscope](https://github.com/datajoint/element-miniscope)
+  + workflow-miniscope: [https://github.com/datajoint/workflow-miniscope](https://github.com/datajoint/workflow-miniscope)
+
+## Sensory Stimulation
+
+### Visual 
++ Example pipeline: [https://github.com/datajoint-catalog/djcat-visual-stimuli](https://github.com/datajoint-catalog/djcat-visual-stimuli)
++ Example pipeline: [https://github.com/datajoint-catalog/djcat-RET1](https://github.com/datajoint-catalog/djcat-RET1)
++ Allen Institute: [https://observatory.brain-map.org/visualcoding/](https://observatory.brain-map.org/visualcoding/)
++ odML - Terminologies: [http://portal.g-node.org/odml/terminologies/v1.0/stimulus/grating.xml](http://portal.g-node.org/odml/terminologies/v1.0/stimulus/grating.xml)
+
+
+## Behavior Tracking
+
+DeepLabCut (DLC) uses TensorFlow machine learning to estimate animal pose or pupillometry from (a) raw video data and (b) user-generated example frames. Typical uses include estimating the location of each ear and the tail base of a mouse to determine both where it was in a maze and the direction it was facing. 
+
+The [repository](https://github.com/DeepLabCut/DeepLabCut) is maintained by Mackenzie Mathis’s team and is quickly becoming a standard in the field. The same process can be used for any species.
+
++ [DLC YouTube](https://www.youtube.com/channel/UC2HEbWpC_1v6i9RnDMy-dfA) covers installation and examples from the Mathis team.
++ DLC pipeline used in Mackenzie Mathis lab: https://github.com/MMathisLab/DataJoint_Demo_DeepLabCut
++ Existing DataJoint repository shows DLC within a workflow:
+  + [https://github.com/datajoint-company/dj-imaging](https://github.com/datajoint-company/dj-imaging) (proprietary)
+
+Treadmill data, including speed and direction, can show an animal’s activity during an experimental session. Head-fixed mice [can be trained](https://www.researchgate.net/publication/272381533_Acute_two-photon_imaging_of_the_neurovascular_unit_in_the_cortex_of_active_mice/figures?lo=1) to use a [spherical treadmill](https://sphericaltreadmill.com/spherical-treadmill-mice-small-animals.aspx) during calcium imaging, optionally with virtual reality [A](https://www.phenosys.com/products/virtual-reality/) & [B](https://www.youtube.com/watch?v=PHZytzVk4E0).
+
++ DataJoint Workflow Tools:
+  + element-deeplabcut: [https://github.com/datajoint/element-deeplabcut](https://github.com/datajoint/element-deeplabcut)
+  + workflow-deeplabcut: [https://github.com/datajoint/workflow-deeplabcut](https://github.com/datajoint/workflow-deeplabcut)
+
+## Neurodata Without Borders (NWB) Data Standard
+
+An nwb file is a highly-specified version of an H5DF file. [Schema docs](https://nwb-schema.readthedocs.io/) provide the relevant details for packaging neurobiology data into this format in both raw and processed forms. Generally, one nwb equates to one experimental session. 
```

Comment:
The NWB team may not like this definition. They emphasize that they could implement NWB using a different substrate (e.g. folder-file structures) rather than HDF5.  You can say something like "NWB is a hierarchical data format with standard schemas for neurophysiology data implemented on top of the HDF5 format." -- or something along these lines.

Reply:
Sounds good. I'll change the wording and include this section in the `element-array-ephys` tutorial branch as Kabi requested above.